### ISSUE TITLE
how to generate attention mask seperately?

### ISSUE THREAD

**Issue by Liujingxiu23** [Shivanandroy/simpleT5#52]
I want to  use my own input_embedings(or my own tokenizer system) instead of input_ids, but I do not know how to generate encoder_attention_mask and decoder_attention_mask bymyself, is there any code?

**Comment by vishugupta96** [comment]
facing the same problem not much info is provided on the inner working of this model making customization difficult.

**Comment by DamithDR** [comment]
You may refer his other repository : https://github.com/Shivanandroy/T5-Finetuning-PyTorch
